### PR TITLE
PCCHAIR-45 Intl Setup, App Theme Update & General Cleanup 

### PR DESCRIPTION
**Pull request by darius-calugar**

### CODE REVIEW

**Review by cata-b — APPROVED** [review]
Looks clean

**Review by AndrewBarzu — COMMENTED** [review]
Looks good, commit too big to look over literally everything but I'm counting on you this won't crash develop ;)

**Review by AndrewBarzu — APPROVED** [review]

File: lib/modules/auth/widgets/login.widget.dart [hunk 1]
```diff
@@ -66,81 +45,93 @@ class LoginState extends State<Login> {
...
+
+  void login(AuthProvider authProvider) {
+    if (_formKey.currentState?.validate() ?? false) {
+      authProvider
+          .login(emailController.value.text, passwordController.value.text)
+          .whenComplete(() {
+        ScaffoldMessenger.of(context).showSnackBar(
+          SnackBar(
+            backgroundColor: authProvider.error == null
+                ? ThemeConfig.of(context)!.successColor
```
Comment (AndreiPetrusMihai) [inline lib/modules/auth/widgets/login.widget.dart:127]: context ii accesibil inafara metodei build?

Comment (darius-calugar) [reply lib/modules/auth/widgets/login.widget.dart:127]: In StatessWidgets ii accesibil doar in build.
In StafefulWidgets ii accesibil in toata clasa de State.

**Pull request merged**